### DISCUSSION TITLE
apache HTTP server has stopped working

### DISCUSSION THREAD

**Discussion by bshf** [leokhoa/laragon#840] [Q&A]
using laragon 6, i start the laragon and never stop it. everything works fine for more than 2 days, then i got this warning
![apache stop working](https://github.com/user-attachments/assets/6d2b5af6-61c2-48aa-b556-f36457412c5f)

  Any suggestions or guesses why this happened? is there something i should do about it?
  
 Thanks

**Comment by rescenic** [comment]
Try reinstalling vcredist.
